dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp

```diff
@@ -337,6 +338,9 @@ void DAGStorageInterpreter::executeImpl(DAGPipeline & pipeline)
     executeCastAfterTableScan(remote_read_streams_start_index, pipeline);
     recordProfileStreams(pipeline, table_scan.getTableScanExecutorID());
 
+    executeGeneratedColumnPlaceholder(generated_column_infos, log, pipeline);
```

Comment:
Don't need generate column place holder for remote table scan? Because the table scan in remote will generate it, or you need to avoid push down generated column to remote side.

Reply:
Done